src/TGameDetails.h

```diff
@@ -0,0 +1,542 @@
+ *           "\n\n"
+ *           "A powerful character customization system allows you to choose your deity –or "
+ *           "fully disregard the gods– and join one of the player-run realms that govern the "
+ *           "land to explore a breathing world, delve into the secrets of the oceans, shape "
+ *           "your legacy, craft forgotten marvels for you –or your allies– and fight for faith, "
+ *           "glory or coin."
+ */
+             qsl("El mud Español gratis con más de 20 años de historia."
+                 "\n\n"
+                 "Reinos de Leyenda toma lugar en el siempre cambiante mundo de Eirea, devastado por "
+                 "las intrigas de los dioses tras más de un millar de años de desprecio y cruenta "
+                 "guerra entre sus fanáticos peones mortales."
+                 "\n\n"
+                 "La historia se escribe día a día, tomando en consideración las elecciones de los "
+                 "jugadores para decidir las consecuencias irreversibles de este conflicto "
+                 "imperecedero."
+                 "\n\n"
+                 "Éste es un MUD con PvP que permite al jugador establecer cuánto quiere arriesgar "
+                 "al morir: a más riesgo, más gloria ganará por sus heroicidades. La interpretación "
+                 "(Rol) no está obligada, pero si recompensada con habilidades especiales -no "
+                 "orientadas al combate- y tesoros únicos."
+                 "\n\n"
+                 "El detallado creador del juego te permitirá elegir tu deidad -o renegar "
+                 "completamente de los dioses- y unirte a uno de los reinos que los jugadores se "
+                 "encargan de gobernar para explorar un mundo viviente, sumergirte en los misterios "
+                 "del océano, dar forma a tu legado, forjar maravillas olvidadas para ti -o tus "
+                 "aliados- y luchar por fe, gloria o dinero.")},
+
+            {qsl("Fierymud"),
+             qsl("fierymud.org"),
+             4000,
+             false,
+             qsl("<a href='https://www.fierymud.org/'>https://www.fierymud.org</a>"),
+             qsl(":/icons/fiery_mud.png"),
+             qsl("The original vision of FieryMUD was to create a challanging MUD for advanced "
+                 "players. This new reborne Fiery is a hope to bring back the goals of the past by "
+                 "inflicting certain death on unsuspecting players. FieryMUD will continue to grow "
+                 "and change through the coming years and those players who seek challenge and "
+                 "possess imagination will come in search of what the 3D world fails to offer them.")},
+
+            {qsl("Mudlet self-test"),
+             qsl("mudlet.org"),
+             23,
+             false,
+             qsl("<a href='https://www.mudlet.org'>www.mudlet.org"),
+             qsl(":/icons/generic_mud.jpg"),
```

Comment:
```suggestion
             QString(), // intentionally not (very) visible yet as it needs to be documented first
```

Reply:
As I say though - that is going to make the icon generator put out that error message and it messes with the navigation with arrow keys which ought to be noticed by those with concerns for **a11y**.

Reply:
I know. But this has been the case for a long while and has not been an issue. Observe how nobody complained or noticed.

We should only make this available when it's polished and documented.